### DISCUSSION TITLE
全球最大 3D 列印工程！中國計劃在青藏高原透過 AI 建造 3D 列印大壩

### DISCUSSION THREAD

**Discussion by Jay-722** [1101-GS4719-C/Python-Community-GS4719-C#604] [General]
**據報導，中國正在利用 AI 人工智慧技術，將青藏高原大壩工程（羊曲水電站）變成一個完全不需工人協助建造的全球最大 3D 列印工程。預計該水電站將在 2024 年完成，屆時能每年提供河南省 50 億瓩時（kWh）的電力。**

什麼都挖土機啥的都是用無人的，很酷，再加上3D列印技術，如果真的蓋好將會是很大的突破

全文 : https://technews.tw/2022/05/14/china-eyes-ai-powered-3d-printed-dam-on-tibetan-plateau/

**Comment by HsuChen88** [comment]
我之前看到這個計畫好像花了很多人民幣..
但它也含有綠能環保的概念

**Comment by HsuChen88** [comment]
如此大的一項建設才在去年開工，
竟然在2024年即將完成，
可見自動化確實減少了很多工程的時間

**Comment by cordydeng** [comment]
之前有看過3D列印出房子，但像這種超大規模的設施，還真的無法想像要怎麼只透過3D列印技術來完成

**Comment by henrywen547** [comment]
2024年真的完成的話，那AI 3D列印的效率真的很高呢!只是完全不須工人也是讓我有點怕怕的

**Comment by jcpan821** [comment]
好難想像水壩這樣蓋

**Comment by 11timyyy** [comment]
似乎是省時又省力，只不過品質希望也能保持是最好的

**Comment by 110409538** [comment]
如果是用AI監控及建造，那品質一定可以管控得更好吧

**Comment by leon10456** [comment]
省時省力的前提是不偷工減料維持安全  如果可以那會是很大的突破
